source/manual/development-pipeline.html.md

```diff
@@ -55,6 +55,12 @@ We've got [guidelines on merging of Pull Requests](/manual/merge-pr.html).
 
 Code that is merged to `master` is tested again on CI. This is because the `master` branch may have changed since the tests last ran on the PR. If the tests on `master` pass, Jenkins pushes a `release_123` git tag to GitHub.
 
+> **WARNING**: some applications have Continuous Deployment enabled, which means the deployment process is fully automated. You should do any manual testing with [a temporary, branch deployment](#branch-deploy-review), before you merge.
+>
+> - Check the notes in the [Release app][release] to see if Continuous Deployment is enabled.
+> - You should still monitor the progress of your deployment in the [Release app][release].
```

Comment:
Just wondering why someone would need to monitor the progress of the deployment? Is it because this is still an "alpha" feature and it might break? What are people looking for?

Reply:
> "alpha"

"beta" - we skipped alpha 'cause we know we want it ;-)

Yeah, that's basically it. The pipeline also has no notification mechanism, so if some code does actually cause a critical issue, the pipeline will fail without anyone knowing, unless they actually check their release made it to production.

I've been thinking about ways to address this. I don't think notifications are a good idea, as that's just a load of noise whatever medium it's on. As we start to switch everything over, I think the deploy badger will fulfil the role of notifying us about undeployed PRs, and I also some people will naturally want to double-check anyway.

Reply:
Yup - might be good to add to that point in the docs, so people understand that they are looking for i.e. deployments actually succeed. But yeah +1 of deploy badger - make sense, "only notify us if something is wrong".